tests/integration.rs

```diff
@@ -3469,6 +3470,44 @@ mod parallel {
         handle_child_output(r, &output);
     }
 
+    #[test]
+    #[cfg(not(feature = "acpi"))]
```

Comment:
@MrXinWang is this test gonna run? I'm asking because if you look at the default features for the project, `acpi` is part of the list.

Reply:
Ahhh yes my bad, this gate should be moved. Thanks for the careful review! I've repushed the branch for @jongwu

Reply:
saw a `test parallel::test_pmu_on ... ok` in the CI log and all current test cases passed.